cmd/users.go

```diff
@@ -430,10 +427,10 @@ var addUserToOrganizationCmd = &cobra.Command{
 	},
 }
 
-var RemoveUserFromOrganization = &cobra.Command{
-	Use:     "user",
-	Aliases: []string{"u"},
-	Short:   "Remove a user to an Organization",
+var RemoveAdministratorFromOrganizationCmd = &cobra.Command{
```

Comment:
All the `Removes` start with an uppercase `R` can these be changed to lowercase to reflect the styling of other commands. It also means it won't be an exported value for no reason.